x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -0,0 +1,126 @@
+[role="xpack"]
+[[fips-140-compliance]]
+== FIPS 140-2
+
+The Federal Information Processing Standard (FIPS) Publication 140-2, (FIPS PUB
+140-2), titled "Security Requirements for Cryptographic Modules" is a U.S.
+government computer  security standard used to approve cryptographic modules.
+{es} is FIPS-140 compliant and as such can run in a FIPS-140 enabled JVM. In
+order to set {es} in fips mode, you must set the
+`xpack.security.fips_mode.enabled` to `true` in `elasticsearch.yml`
+
+For {es}, FIPS 140-2 compliance is ensured by
+
+- Using FIPS approved / NIST recommended cryptographic algorithms.
+- Delegating the implementation of these cryptographic algorithms to a NIST
+  validated cryptographic module (available via the Java Security Provider
+  in use in the JVM).
+- Allowing the configuration of {es} in a FIPS 140-2 compliant manner, as
+  documented below.
+
+IMPORTANT: Only {es} 6.4.0 and later can be run in a FIPS 140-2 enabled JVM
```

Comment:
I'm not sure we need this type of statement. @lcawl what is your take?

Reply:
I think this support statement is redundant, since this page will exist only in 6.4 and later documentation.